### ISSUE TITLE
Wizard sub-steps expanded vs hidden

### ISSUE THREAD

**Issue by matthewcarleton** [patternfly/patternfly-design#769]
@mcarrano and I had a discussion around this [here](https://github.com/patternfly/patternfly-next/pull/2186#discussion_r317624093)

TL:DR
Should wizard sub steps be hidden until the user navigates to the parent step? 
Standard conventions dictate that they should be hidden. If we look at our navigation component we see the sub items hidden. This is for both space saving as well as cognitive load. 
Where the Wizard differs is this: The purpose with a wizard is to allow the user to understand all the steps they need to complete so this could justify show all sub steps. The question is do we still want to consider if saving space and overloading the user are valid reasons to hide them.
Sub steps by definition should be secondary to the parent step so they parent step should indicate the amount of effort required. If a sub step is needed to be seen all the time it should be a parent level step or incorporated into an existing parent level step.

This is what needs to be decided.

**Comment by maryshak1996** [comment]
WIP proposal: https://marvelapp.com/prototype/142h0f9e

**Comment by mcarrano** [comment]
Thanks for you work on this @maryshak1996 ! I have opened https://github.com/patternfly/patternfly/issues/3904 to begin core implementation.
